docs(preamble): :memo: add the assumptions and "who you are" section (#13)
rostools/tutorial-github-intro: preamble/syllabus.qmd

-   Item 1

## Is this for you? {#sec-is-it-for-you}

To help manage expectations and develop the material for this tutorial,
we make a few possible assumptions about *who you are* as a participant
in the tutorial, such as one or more of the following:

-   You want a gentle introduction to what GitHub is and how to use it.
-   You mostly work alone or maybe irregularly with someone else on your
    project.

> -   You are assigned tasks on a project that others are also involved
>     in, but the tasks don't overlap with others, require direct
>     contributions from others, or don't share the same files as others.

Reviewer: I feel like this overlaps a bit with the first two points? Could it be removed?
```suggestion
```

-   You write text in documents as a main activity of your work.

> -   You are able to and are comfortable enough to only work on files

Reviewer: ```suggestion
-   You are able to and are comfortable enough with only working on files
```

    through a web browser interface (like GitHub's interface).

While we have these assumptions to help focus the content of the course,
if you have an interest in the tutorial but don't fit any of the

> assumptions, *you are still welcome to attend the tutorial*! We welcome

Reviewer: ```suggestion
assumptions, *you are still welcome to attend *! We welcome
```
